tests/integration/pilot/vm/util.go

```diff
@@ -17,10 +17,11 @@
 package vm
 
 const (
-	DefaultVMImage = "app_sidecar_ubuntu_bionic"
+	// TODO do not merge until I revert back to debian
+	DefaultVMImage = "app_sidecar_centos"
 )
 
 func GetSupportedOSVersion() []string {
 	return []string{"app_sidecar_ubuntu_xenial", "app_sidecar_ubuntu_focal", "app_sidecar_ubuntu_bionic",
-		"app_sidecar_debian_9", "app_sidecar_debian_10"}
+		"app_sidecar_debian_9", "app_sidecar_debian_10", "app_sidecar_centos"}
```

Comment:
nit: Do you think it's better to put the centos version as a suffix to keep the standard with the debian images?

Reply:
:+1: good point, I do think it is

Reply:
I think this one and the default image above has not been updated to be app_sidecar_centos_8. It might fail afterwards.